cmd/kind/kind.go

```diff
@@ -94,17 +94,14 @@ func Run() error {
 
 // Main wraps Run and sets the log formatter
 func Main() {
+	// TODO(bentheelder): replace this with a shimmable logger
 	// let's explicitly set stdout
-	log.SetOutput(os.Stdout)
+	log.SetOutput(os.Stderr)
```

Comment:
possibly, modify the above comment `set stdout` or keep as is?

Reply:
that should say stderr for sure, using stdout was a bug but not related to the rest of this PR ...

Reply:
fixed